datasetinsights/stats/visualization/plots.py

```diff
@@ -370,3 +372,90 @@ def rotation_plot(df, x, y, z=None, max_samples=None, title=None, **kwargs):
         )
     )
     return fig
+
+
+def model_performance_comparison_box_plot(
+    title,
```

Comment:
Why are you fixing title at the first position?

Reply:
Add a default value.